backend/tests/integration/diff/test_diff_merge.py

```diff
@@ -146,11 +146,11 @@ async def test_node_delete_conflict(
         await new_person.save(db=db)
         diff_branch = await create_branch(db=db, branch_name="branch2")
         if delete_on_branch:
-            delete_branch = default_branch
-            update_branch = diff_branch
-        else:
             delete_branch = diff_branch
             update_branch = default_branch
+        else:
+            delete_branch = default_branch
+            update_branch = diff_branch
```

Comment:
the naming and logic was inverted here, which confused me while investigating this test